src/puppetlabs/puppetdb/queue.clj

```diff
@@ -250,21 +250,32 @@
                  file-extension))))
 
 (defn cmdref->cmd [q cmdref]
+  "Returns the command associated with cmdref, or logically false if
+  the file is missing (i.e. someone has deleted it)."
```

Comment:
OK, so @mullr pointed out that the norm in the clojure.core docs is nil or non-nil (I'd remembered something else), and so I agree we should parallel that.